recipes/getversion/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "getversion" %}
+{% set version = "0.5.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cec571c6f2578e3388339c97cf6706d64e877742d040f315206d2f5c09f68c0b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools_scm
+    - six
```

Comment:
Shouldn't these all below go into run?

Reply:
Maybe? So far I've only seen them in `setup.py` rather than the actual python module.

Reply:
The package is a mess :(

Reply:
I was playing with this package locally, and those dependencies do need to be in `host`. And I second Devon that the package is a mess :/